pkg/api/frontendlogging/source_maps.go

```diff
@@ -68,13 +68,16 @@ func (store *SourceMapStore) guessSourceMapLocation(sourceURL string) (*sourceMa
 		return nil, err
 	}
 
-	// determine if source comes from grafana core, look in public build dir
-	if strings.HasPrefix(u.Path, "/public/build/") {
-		return &sourceMapLocation{
-			dir:      store.cfg.StaticRootPath,
-			path:     filepath.Join("build", u.Path[len("/public/build/"):]) + ".map",
-			pluginID: "",
-		}, nil
+	// determine if source comes from grafana core, locally or CDN, look in public build dir on fs
+	if strings.HasPrefix(u.Path, "/public/build/") || (store.cfg.CDNRootURL != nil && strings.HasPrefix(sourceURL, store.cfg.CDNRootURL.String()) && strings.Contains(u.Path, "/public/build/")) {
```

Comment:
this can be simplified to simply store.cfg.GetContentDeliveryURL(..) + "/public/build"    (public/build is also part of CDN path)

Reply:
I considered this, but don't have `hs.License` in scope here to call `store.cfg.GetContentDeliveryURL(..)`. Could add it to `SourceMapStore`  struct, but no longer makes it that simpler. WDYT?